.evergreen/auth_oidc/setup_oidc.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Set up OIDC auth roles.
+ */
+(function() {
+"use strict";
+
+const admin = Mongo().getDB("admin");
+assert(admin.auth("bob", "pwd123"));
+assert.commandWorked(admin.runCommand({createRole: 'test1/readWrite', roles:[{role: 'readWrite', db: "test"}], privileges: []}));
+assert.commandWorked(admin.runCommand({createRole: 'test2/read', roles:[{role: 'read', db: "test"}], privileges: []}));
```

Comment:
Would it make sense to have a mongosh-friendly version of this? It uses Node's `assert` module so `assert.commandWorked` does not exist but we could get the equivalent in mongosh with:

```ts
  assert.deepEqual({ ok: 1 }, admin.runCommand({createRole: 'test1/readWrite', roles:[{role: 'readWrite', db: 'test'}], privileges: []}));
  assert.deepEqual({ ok: 1 }, admin.runCommand({createRole: 'test2/read', roles:[{role: 'read', db: 'test'}], privileges: []}));
```

Reply:
`assert.deepEqual` doesn't work with `mongo`, and it seems `mongosh` is not downloaded by `run-orchestration.sh`, unless I'm missing something?

Reply:
Correct, I was asking if it would be worth it to have a separate potential script, like `setup_oidc_mongosh.js` that could be optionally used. (Since `mongo` is EOL and mongo-orchestration needs to get a legacy binary). 

I just brought it up as a question - but nothing needs to happen here. I just did this in the Node driver for now because we'd prefer to use mongosh: https://github.com/mongodb/node-mongodb-native/pull/3560/files#diff-58d5ff1e7dd36ee2f194daa1bb078a421899232c908c8e9bbe50c5401f0c2d02

Reply:
Perhaps instead we should update `run-orchestration.sh` to install `mongosh` as well, and only use `mongosh` in this instance?  That would make it easier to update other usages of `mongo` in this repo.

Reply:
That sounds like a good idea.

Reply:
Actually, that's a lot more complicated than I thought, given the need to support multiple platforms.  I'll defer to a dedicated effort and add the second script.